src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (isNaN(val)) {
```

Comment:
`_.isNaN` is not the same as `isNaN`, read here: http://underscorejs.org/#isNaN.

We need to make sure this is covering the cases that we want it to cover